lib/ansible/modules/windows/win_audit_policy_system.ps1

```diff
@@ -0,0 +1,129 @@
+#!powershell
+# Copyright (c) 2017 Noah Sparks <nsparks@outlook.com>
```

Comment:
Just being pedantic here, not a showstopper. But the syntax for copyright statement was changed to:
```powershell
# Copyright: (c) 2017, Noah Sparks <nsparks@outlook.com>
# Copyright: (c) 2017, Ansible Project
```

Reply:
Np. I'll change since it looks like there's some new validation it failed for other items.